.github/workflows/ci.yml

```diff
@@ -23,7 +23,8 @@ jobs:
         - 3.5
         - 3.6
         - 3.9
-        - 3.10-dev
+        # Broken b/o https://github.com/pytest-dev/pytest/issues/8539
+        # - 3.10-dev
```

Comment:
Sounds fine to me until this gets resolved.

Reply:
By the way, this would be much better as it's own PR, not mixed into an unrelated PR. Also makes it easier to revert.

Reply:
If you can cherry-pick b3fff905bed994a9aada72565cc29305991c03b5 I think we can accept it immediately and then restart the CI (or just ignore these jobs) for the pending PRs.

Reply:
Hi @henryiii, thanks, I'm not sure how obvious this is in your view, this is only the smart_holder branch. (Pointing out to make sure you're not surprised.)

Reply:
Not super obvious, that explains it; worth cherry-picking and applying to master via a PR, I think.

Reply:
That does help explain the smart-holder PRs. Maybe a labeler or prefix (like `[smart holder]`) or such would help. Or I can just watch for the target branch. :)